src/ui/SimpleFilter/SimpleFilterValues.ts

```diff
@@ -0,0 +1,65 @@
+import 'styling/_SimpleFilter';
+import { IDoneBuildingQueryEventArgs, IQuerySuccessEventArgs } from '../../events/QueryEvents';
+import { ComponentOptions, IFieldOption } from '../Base/ComponentOptions';
+import * as _ from 'underscore';
+import { IGroupByRequest } from '../../rest/GroupByRequest';
+import { QueryBuilder } from '../Base/QueryBuilder';
+import { SimpleFilter } from './SimpleFilter';
+
+export interface ISimpleFilterOptions {
+  field: IFieldOption;
+  maximumNumberOfValues: number;
+}
+
+export class SimpleFilterValues {
+
+  static options: ISimpleFilterOptions = {
+    maximumNumberOfValues: ComponentOptions.buildNumberOption({ defaultValue: 5 }),
```

Comment:
`SimpleFilterValues` is not a component. You do not need to build options like that.

`ComponentOptions.buildNumberOption` is used only to parse the HTML element for a component.